cmd/get.go

```diff
@@ -108,4 +109,6 @@ sudo install -m 755 %s /usr/local/bin/%s
 const arkadeGet = `Use "arkade get TOOL" to download a tool or application:
 
   - kubectl
-  - faas-cli`
+  - faas-cli
+  - kubectx
```

Comment:
One consideration for Windows users is that they will get a .exe appended. We need to configure the struct or extend it so that the code `finalName = finalName + ".exe"` doesn't run for bash scripts such as this one.

Perhaps we can have a field like `NoExtension` which is set to `true`? I'm open to other ideas.

@Waterdrips what do you think?

Reply:
Yeh that would work. I think we need add things we need to get it working now, then change the approach a little down the line if needed (like with the apps at the moment)